### DISCUSSION TITLE
[Feature] Keyboard shortcut to stack selected photos

### DISCUSSION THREAD

**Discussion by gga-kialo** [immich-app/immich#5143] [Feature Request]
### The feature

I use the stack feature often. Being able to select multiple photos and stacking them by say pressing 's' would be very convenient.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by flightmansam** [comment]
Absolutely agreed

**Comment by flightmansam** [comment]
https://github.com/immich-app/immich/pull/5983

**Comment by isaacolsen94** [comment]
This feature would be so helpful! Any update on #5983 possibly being merged?
